Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI.py

```
@@ -0,0 +1,535 @@
+        "AssignedUserID": assigned_user_id,
+        "AIDriverID": ai_driver_id,
+        "PolicyID": policy_id,
+        "Subject": user_name or "",
+        "SecurityLog": security_log,
+        "Status": 1
+    }
+    return payload
+
+
+def test_module():
+    """If API key set → simple GET / with Bearer; else try cookie login."""
+    base_url = (demisto.params().get('url') or '').rstrip('/')
+    api_key = demisto.params().get('api_key') or ''
+    email = demisto.params().get('user_email') or ''
+    password = demisto.params().get('user_password') or ''
+
+    _require_param('url', base_url)
+    s = _new_session()
+
+    if api_key:
+        r = s.get(f'{base_url}/', headers=api_key_headers(api_key), verify=_verify_flag(), timeout=15)
+        if r.status_code in (200, 302):
+            return_results('ok')
+        else:
+            return_error(f'API key test failed: {r.status_code} {_debug_slice(r.text)}')
+    else:
+        _require_param('user_email', email)
+        _require_param('user_password', password)
+        _, _ = login_to_dotnet_api(s, base_url, email, password)  # raises on failure
+        return_results('ok')
+
+
+def shadowx_submit_task_command():
+    args = demisto.args()
+    base_url = (demisto.params().get('url') or '').rstrip('/')
+    api_key = demisto.params().get('api_key') or ''
+
+    # Required arg
+    log = args.get('log')
+    if not log:
+        return_error("'log' is required.")
+
+    payload = _build_task_payload_from_args(args)
+    wait_seconds = int(args.get('wait_seconds', 0))
+
+    s = _new_session()
+
+    if api_key:
+        # API mode (Bearer)
+        submit_res = submit_with_api_key(s, base_url, api_key, payload)
+        if submit_res.status_code != 200:
+            return_error(f"Task submit failed (API key): {submit_res.status_code} {_debug_slice(submit_res.text, 600)}")
+
+        submit_json: Dict[str, Any] = {}
+        try:
+            submit_json = submit_res.json()
+        except Exception:
```

Silently passing on all exceptions with `except Exception: pass` is risky. If JSON parsing fails, the error is hidden, and the command proceeds with an empty dictionary, which can lead to confusing behavior. At a minimum, a debug log should be added.